src/Controllers/Slack/ConversationController.php

```diff
@@ -216,7 +216,8 @@ private function determineEventIntent(SensorEvent $e)
         $intent = null;
         switch (true) {
             case $e instanceof SlackInteractiveMessageEvent:
-                $intent = new Intent($e->getCallbackId(), $e, 1);
```

Comment:
The only place $e should appear is in the listen function. handleOngoingConversation, handleNewConversation and handleNothingMatched should only deal with an utterance.